substrate/frame/staking-async/src/pallet/mod.rs

```diff
@@ -868,6 +886,9 @@ pub mod pallet {
 			if let Some(x) = self.max_nominator_count {
 				MaxNominatorsCount::<T>::put(x);
 			}
+			if let Some(x) = self.unbonding_queue_config {
```

Comment:
```suggestion
			UnbondingQueueParams::<T>::put(self.unbonding_queue_config);
```

Reply:
Changed in [aa3d40c](https://github.com/paritytech/polkadot-sdk/pull/8298/commits/aa3d40c2cb608696e9669c2a5f1bea022fd4848d).